doc/CRD-generator.md

```diff
@@ -1,27 +1,42 @@
 # CRD generation
 
+The CRD Generator annotation processing tool (APT) (`io.fabric8:crd-generator-apt`) and its API (`io.fabric8:crd-generator-api`) are being deprecated and will eventually be removed once we offer a complete replacement for all users.
+
+As a replacement, we're currently providing a new version of the API in the `io.fabric8:crd-generator-api-v2`and a few tools to be able to leverage it in your projects.
+
+The following list contains the available tooling:
+- CRD Generator Maven Plugin: A Maven plugin that generates CRDs during the build process.
+- CRD Generator CLI tool: A CLI tool that generates CRDs when executed.
+
 ## Quick start
 
-Import the Annotation Processor into your build.
+Add the CRD Generator plugin to your project or use the CLI tool:
```

Comment:
```suggestion
Add the CRD Generator plugin to your project:
```

Reply:
In my opinion in the quickstart we should focus on the maven and gradle plugin. Compared to those tools the CLI will be less common and you already provided a complete list of the tooling in the beginning.